.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.0
```

Comment:
Just out of curiosity, why do we need different gemsets here? All we do is use a number of standard Ruby gems.